### DISCUSSION TITLE
Bank Presets

### DISCUSSION THREAD

**Discussion by CombatBorn** [runelite/runelite#14915] [Ideas]
**I made some images to make the idea easier to understand:**

The idea is simplicity. A simple way to **create**, **name**, **save**, and **search** for items of a custom user made preset
![idea1](https://user-images.githubusercontent.com/6846423/166289712-97237600-cfce-4fde-b473-4627c45b1d9f.png)

Clicking Search would highlight which items from the preset you have correct. 
 Green = Correct Item in slot
 Orange = You own the item in bank
 Red = You don't own the item
![idea2](https://user-images.githubusercontent.com/6846423/166289726-44baa425-abc1-4f87-a0ca-aae79e069999.png)

- The save button would simply look at the items that are already in your inventory/equipment slots and store the data locally.

- Not sure what would be an easy way to change a preset icon.

**Comment by Enriath** [comment]
This is just `Inventory Setups`, a plugin already in the hub. It's not the best, but it does what's asked for here and a bit more.

**Reply by CombatBorn** [reply]
Thanks! I believe my design is a bit more user friendly, but this works great! Really appreciate it.
